Modules and packages/Import module 2/task.md

```diff
@@ -0,0 +1,15 @@
+## Import module 2
+
+You can import not only functions: you can import classes or even other modules. It is customary but not required to place all 
+import statements at the beginning of a module.
+
+You can find out more about modules in Python by reading [this section](https://docs.python.org/3/tutorial/modules.html) of The Python Tutorial.
```

Comment:
You can learn more about modules in Python by reading [this section](https://docs.python.org/3/tutorial/modules.html) of The Python Tutorial.